lib/matplotlib/backend_bases.py

```diff
@@ -2559,6 +2674,120 @@ class NonGuiException(Exception):
     pass
 
 
+class WindowEvent(object):
+    def __init__(self, name, window):
+        self.name = name
+        self.window = window
+
+
+class WindowBase(cbook.EventEmitter):
+    """The base class to show a window on screen.
+
+    Parameters
+    ----------
+    title : str
+        The title of the window.
+    """
+
+    def __init__(self, title, **kwargs):
+        super(WindowBase, self).__init__(**kwargs)
```

Comment:
devil's advocate: you are documenting that it accepts a title argument, and you even have a method for setting a title, but they do nothing. It is entirely misleading to me.